jekyll/_cci2/[WIP]runner-overview.adoc

```diff
@@ -0,0 +1,100 @@
+
+== Runner operation
+
+Once CircleCI runner is installed, the runner polls circleci.com for work, runs jobs, and returns status, logs, and artifacts to CircleCI. When the runner is not running a job, it will auto-update itself when a new version is released.
+
+The runner consists of two components: the Launch Agent and the Task Agent.
+
+* Launch Agent (launch-agent) - manages gathering the information required to run a task (defined as a parallel run of a job) while also downloading and launching a Task Agent process.
+* Task Agent (task-agent) - handles running a task retrieved and configured by the Launch Agent.
+
+The system has been designed to allow administrators to configure the task-agent to run with a lower level of privileges than the launch-agent. Any user who is able to execute a job will be able to gain the same privileges as task-agent. The instructions below are the recommended deployment which follows this approach (Launch Agent will run as root, but Task Agent will run as circleci).
+
+== CircleCI runner limitations
+
+There are a few limitations you should be aware of when using CircleCI runner:
+
+The following platforms are currently supported:
+
+* Intel + Ubuntu
+* Arm64 + Ubuntu
+* Intel + macOS
+
+For more information on additional platforms that have been tested, but are not supported, and platforms that have not been tested, please refer to the Runner Installation [add link here when ready] page.
+
+The following features are not yet available, but may be in the future:
+
+* Rerun with SSH
+* Test splitting
+* add_ssh_keys
+
+== Public repositories
+
+CircleCI runner is not recommended for use with public projects that have the "Build forked pull requests" setting enabled. In this case, a malicious actor may alter your machine or execute code on it by forking your repository, committing code, and opening a pull request. Untrusted jobs running on your CircleCI runner pose significant security risks for your machine and network environment, especially if your machine persists its environment between jobs. Some of the risks include:
+
+* Malicious programs running on the machine.
+* Escaping the machine's runner sandbox.
+* Exposing access to the machine's network environment.
+* Persisting unwanted or dangerous data on the machine.
+
+== Prerequisites
+
+In order to use CircleCI runner, you must:
+
+* Be on the Scale Plan.
+* Use CircleCI Cloud.
+* Have experience managing your infrastructure.
+* Install [add link here when ready] CircleCI runner.
+
+== Referencing your runner on a job
+
+After setting up the runner, you will need to reference it on a job by setting some fields in a special way in your `.circleci/config.yml` file. The fields you must set for a specific job to run using your runner are:
+
+* `machine: true`
+* `resource_class: your-namespace/your-resource` 
```

Comment:
Should we introduce what a namespace is somewhere in this doc? For a new user, it's not clear what this is or where to go to learn what it is.

Reply:
Added note about namespace.